common/tools/dev-tool/src/commands/admin/migrate-package.ts

```diff
@@ -352,18 +356,18 @@ async function upgradeTypeScriptConfig(tsconfigPath: string): Promise<void> {
   tsConfig.compilerOptions.moduleResolution = "NodeNext";
   tsConfig.compilerOptions.rootDir = ".";
   tsConfig.include = [
-    "./src/**/*.ts",
-    "./src/**/*.mts",
-    "./src/**/*.cts",
-    "./samples-dev/**/*.ts", // TODO: Check if samples-dev is needed
-    "./test/**/*.ts",
+    "src/**/*.ts",
```

Comment:
would `src/**/*.{ts,cts,mts}` work for TS?

Reply:
Ya it could work! I think the linter rule we have in place will still complain about that format but I can see if I can update the rule in a future commit